Sources/RTMP/RTMPStream.swift

```diff
@@ -430,17 +430,14 @@ open class RTMPStream: NetStream {
     open func createMetaData() -> ASObject {
         metadata.removeAll()
         #if os(iOS) || os(macOS)
-        if let _: AVCaptureInput = mixer.videoIO.input {
```

Comment:
This is not good. This code will send video and audio metadata always. I don't want send.

Reply:
Please override your code.
```
class YourRTMPStream: RTMPStream {
    override func createMetadata() -> ASObject  {
           ...
    }
}
```

Reply:
I see, I didn't think of the case that a user might want to only send video or audio but not both.
I made a new class where user can set flags to specify what kind of sampleBuffer they intend to send.
Hope this is okay, let me know if more changes are needed. Thanks!

Reply:
Ah, I have the issue that I can't use this in my application code because mixer.videoIO and mixer.audioIO have 'internal' protection level, so my app cannot access them. Is there any way to send metadata for apps that use the appendSampleBuffer method?